pages/common/dot.md

```diff
@@ -1,12 +1,25 @@
 # dot
 
-> A command-line tool to produce layered drawings of directed graphs.
-> More information: <https://www.graphviz.org/pdf/dotguide.pdf>.
+> Render an image of a `linear directed` network graph from a `graphviz` file.
+> Layouts: `dot`, `neato`, `twopi`, `circo`, `fdp`, `sfdp`, `osage` & `patchwork`.
+> More information: <http://graphviz.org/doc/info/command.html>.
 
-- Render an image file and determine output filename based on input filename and selected format:
+- Render a `png` image with a filename based on the input filename and output format (uppercase -O):
 
-`dot -Tpng -O {{path/to/file.dot}}`
+`dot -T png -O {{path/to/input.gv}}`
```

Comment:
```suggestion
`dot -T {{png}} -O {{path/to/input.gv}}`
```
png is a fixed value, but still an argument. And I think highlighting it can't hurt